lms/djangoapps/grades/api/views.py

```diff
@@ -226,3 +235,104 @@ def get(self, request, course_id, **kwargs):
         if isinstance(course, Response):
             return course
         return Response(GradingPolicySerializer(course.raw_grader, many=True).data)
+
+
+@view_auth_classes()
+class AdditionalGradeReport(GenericAPIView):
```

Comment:
[As noted previously](https://github.com/edx-olive-oc/edx-platform/pull/5#pullrequestreview-141877549), I recommend moving all changes related to these additional reports into their own django app, e.g. `lms/djangoapps/grades_report`.  This makes these changes more maintainable, and allows us to upstream them as a separate django plugin app if needed.